docs/html5sortable.js

```diff
@@ -1010,6 +1025,25 @@ var sortable = (function () {
                           originalTarget: target
                       }
                   }));
+                  addEventListener(sortableContainer, 'dragleave', function (e) {
+                      var outTarget = e.releatedTarget || e.fromElement;
```

Comment:
This definitely has to be fixed to `relatedTarget` (there is a typo). `e.fromElement` is for very old browsers, but maybe we just add a comment `// e.fromElement for very old browsers, similar to relatedTarget`.

Maybe you can also add a comment above `// TODO: rename outTarget to be more self-explanatory`-

Basically I think it is the parent element (not sortable) of the sortable that the item was in. We are checking that it actually is moved to a different sortable and does not stay within.